### DISCUSSION TITLE
<title>jailbreak  disabled iPhone 12

### DISCUSSION THREAD

**Discussion by Imsolost63** [community/community#128290] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

is there a way rot jailbreak a disabled iPhone 12

**Comment by ebndev** [comment]
Hi @Imsolost63,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
